.releaserc.json

```diff
@@ -1,12 +1,5 @@
 {
-  "branches": [
-    "main",
-    {
-      "channel": "next",
-      "name": "(f|b|c)/*",
-      "prerelease": "beta-<%= (/^\\w+-\\d+/.exec(name.substr(2)) || [])[0] %>"
-    }
-  ],
+  "branches": ["main"],
```

Comment:
No need to add branches but just the @open-turo/semantic-release-config as an extends like in here https://github.com/open-turo/renovate-config/blob/main/.releaserc.json